guides/promotions/promotion-rules.md

```diff
@@ -0,0 +1,111 @@
+# Promotion rules
+
+The `Spree::PromotionRule` model sets a rule that determines whether a promotion
+is eligible to be applied. Promotions may have no rules or many different rules.
+
+An example of a typical promotion rule would be a minimum order total of $75
+USD or that a specific product is in the cart at checkout.
+
+Many conventional types of promotion rules are included by default. Multiple
+promotion rules can exist on each promotion, although no rules are required.
+
+# Available promotion rule types
+
+The following classes are [subclasses of the `Spree::Promotion::Rules`
+model][promotion-rules]:
+
+- `FirstOrder`: Eligible for a customer's first order only.
+- `FirstRepeatPurchaseSince`: Eligible for a customer's first repeat purchase
+  since a specified date.
+- `NthOrder`: Eligible for a customer's *n*th order only.
+- `ItemTotal`: Eligible if the order total is less than or greater than a
+  specified amount.
+- `OneUsePerUser`: Eligible for use one time for each user.
+- `Product`: Eligible for specified products only. 
+- `OptionValue`: Eligible for specified variants (product option values) only.
+- `Taxon`: Eligible for products with specified taxons.
+- `User`: Eligible for specified users.
+- `UserRole`: Eligible for users with the specified user role.
+- `UserLoggedIn`: Eligible for users who are logged in.
+
+<!-- TODO:
+  It may be useful to link to option values documentation and taxons
+  documentation here, unless we explain what they are further in-line.
+  Once that documentation is merged, we can add those link.
+-->
+
+[promotion-rules]: https://github.com/solidusio/solidus/tree/master/core/app/models/spree/promotion/rules
+
+## Eligibility
+
+Note that whenever an order, line item, or shipment with a promotion adjustment
+on it is updated, the [eligibility][eligibility] of the promotion is re-checked
+and the promotion actions are re-applied.
```

Comment:
> the promotion actions are re-applied.

Might be a bit misleading. The action isn't really re-applied. The amount will be re-calculated and the eligibility of the adjustment will be changed based on the eligibility and if the amount becomes $0. But I'm pretty sure all of the logic in the action's `perform(options = {})` method won't be re-run.